source/code/plugin/filter_inventory2mdm.rb

```diff
@@ -0,0 +1,261 @@
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+
+# frozen_string_literal: true
+
+module Fluent
+    require 'logger'
+    require 'json'
+    require_relative 'oms_common'
+
+	class Inventory2MdmFilter < Filter
+		Fluent::Plugin.register_filter('filter_inventory2mdm', self)
+		
+		config_param :enable_log, :integer, :default => 0
+        config_param :log_path, :string, :default => '/var/opt/microsoft/docker-cimprov/log/filter_inventory2mdm.log'
+        config_param :custom_metrics_azure_regions, :string
+        
+        @@node_count_metric_name = 'nodesCount'
```

Comment:
I am thinking if we should name these as 'numPods' & 'numNodes'

Reply:
Why? I think nodeCount and podCount is just fine

Reply:
That seems to be the convention for 'count' metrics. But i am okay to leave it as is until somebody tells so :)